spec/views/content_assets/index.html.erb_spec.rb

```diff
@@ -2,14 +2,10 @@
 
 RSpec.describe "content_assets/index", type: :view do
   before(:each) do
-    assign(:content_assets, [
-      ContentAsset.create!(
-        title: "Title",
-      ),
-      ContentAsset.create!(
-        title: "Title",
-      ),
-    ])
+    @content_asset = FactoryBot.create(:content_asset)
```

Comment:
Can we move these variables into let definitions
https://relishapp.com/rspec/rspec-core/v/2-11/docs/helper-methods/let-and-let

Reply:
Turns out the rendered view requires the instance variable `@content_assets` otherwise it'll fail with a nil object error.
Ignore that request